lib/config/presets/internal/regex-managers.ts

```diff
@@ -14,4 +14,16 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  helmChartYamlAppVersions: {
+    description: 'Update `appVersion` value in helm chart Chart.yaml.',
```

Comment:
```suggestion
    description: 'Update `appVersion` value in Helm chart Chart.yaml.',
```

Helm is a proper noun, so we should capitalize it.